testsuite/MDAnalysisTests/analysis/test_align.py

```diff
@@ -118,6 +118,21 @@ def test_rms_fit_trj(self):
         self._assert_rmsd(fitted, 0, 6.929083044751061)
         self._assert_rmsd(fitted, -1, 0.0)
 
+    def test_AlignTraj(self):
```

Comment:
can you also test that all warnings/exceptions are raised. The best thing is to make sure with coverage that the whole class is covered.